cpp/modmesh/view/R3DWidget.hpp

```diff
@@ -60,6 +60,45 @@ class ROrbitCameraController
     using Qt3DExtras::QOrbitCameraController::QOrbitCameraController;
 }; /* end class ROrbitCameraController */
 
+class RCustomCameraController
```

Comment:
Even though this is prototype code, we may use a descriptive name for the class.  What name would be informative?

Reply:
hmm, I don't know, I think it depends on the behavior of the camera? like orbital or FPS

Reply:
How would you like the camera to behave (move)?